apps/src/codebridge/FileBrowser/styles/filebrowser.module.scss

```diff
@@ -4,7 +4,6 @@
   height: 100%;
   background-color: $darkest_gray;
   color: $white;
-  grid-area: file-browser;
```

Comment:
neither of these should have grid-areas tacked onto them because the outer layout grid will pick them up and get confused.

I need to research a more dynamic way to add in these names so we don't have issues in the future.